src/Microsoft.Diagnostics.Runtime/Native/NativeRuntime.cs

```diff
@@ -261,6 +275,95 @@ private void InitThreads()
             _threads = threads.ToArray();
         }
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
+        {
+            if (!this.ClrInfo.DacInfo.PlatformAgnosticFileName.Contains("mrt100"))
+                throw new ClrDiagnosticsException("Serialized exceptions are only available for a Native data target");
+
+            if (_sosNativeSerializedExceptionSupport == null)
+                throw new ClrDiagnosticsException("This version of mrt100 is too old.", ClrDiagnosticsException.HR.DataRequestError);
```

Comment:
As mentioned above, this should be `return new ClrException[0];`.  In cases where data is not available we return an empty list instead of throwing.